package.json

```diff
@@ -9,7 +9,8 @@
     "README.md"
   ],
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "mvnbuild": "gulp bundle",
+    "mvntest": "gulp test"
```

Comment:
Newer version of NPM complains about this duplicate key. The second version was the one actually being used anyway.